Utilities/DataFlow/include/DataFlow/SubframeMetadata.h

```diff
@@ -0,0 +1,24 @@
+#ifndef SUBFRAMEMETADATA_H
```

Comment:
This file can be moved to `DataFormats/Headers`, by this the dependencies on the module `DataFlow` can be avoided